packages/react-pose-core/src/components/PoseComponent.tsx

```diff
@@ -89,7 +89,7 @@ class PoseComponent extends React.PureComponent<PoseComponentProps> {
     const poseList: string[] = Array.isArray(pose) ? pose : [pose];
 
     Promise.all(poseList.map(key => key && this.poser.set(key))).then(
-      () => onPoseComplete && onPoseComplete()
+      () => onPoseComplete && onPoseComplete(pose)
```

Comment:
Very nice idea! Would it be better here to pass `key`? That way its simpler in `onPoseComplete` to compare against a specific pose

Reply:
there might be multiple keys (see `Promise.all` usage), so we have to pass either `pose` or `poseList` here

Reply:
Ah of course I misread my own code